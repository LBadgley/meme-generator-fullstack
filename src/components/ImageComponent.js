import React from 'react';
import PropTypes from 'prop-types';

export default function ImagePicker({ image, handleChange }) {
  return (
    <>
      <p>Image: </p>
      <input name="image" type="text" value={image} onChange={handleChange} />
    </>
  );
}

ImagePicker.propTypes = {
  image: PropTypes.string.isRequired,
  handleChange: PropTypes.func.isRequired
};
