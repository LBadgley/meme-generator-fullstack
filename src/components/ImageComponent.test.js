import React from 'react';
import { shallow } from 'enzyme';
import ImageComponent from './ImageComponent';

describe('image component tests', () => {
  it('renders the image component', () => {
    const wrapper = shallow(<ImageComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
