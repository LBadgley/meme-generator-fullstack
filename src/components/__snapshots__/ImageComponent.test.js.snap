// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`image component tests renders the image component 1`] = `
<Fragment>
  <p>
    Image: 
  </p>
  <input
    name="image"
    type="text"
  />
</Fragment>
`;
